### PR TITLE
Change SPs Status: RFC > Obsolete

### PR DESCRIPTION
**Pull request by Gala-G**
MIP10
MIP7
MIP39
MIP40
MIP41

@patrick-j-govalpha

### CODE REVIEW

**Review by LongForWisdom — CHANGES_REQUESTED** [review]
@Gala-G, can you remove all the extra whitespace changes from this PR? It makes it difficult to track what the actual changes are, and is bad practice in a general sense. 

This changelist should only consist of changes to the preamble. Modifying the status, and adding the extra information.

File: MIP41/MIP41c5-Subproposals/MIP41c5-SP2.md
Comment (LongForWisdom) [inline MIP41/MIP41c5-Subproposals/MIP41c5-SP2.md:1]: This file should not be changed

**Comment by LongForWisdom** [comment]
You missed this one:
https://mips.makerdao.com/mips/details/MIP14c2SP5

**Comment by LongForWisdom** [comment]
You will need to check that these proposals shouldn't have been set to 'Accepted' status. I believe most of the MIP10 ones ended up passing.

**Comment by ghost** [comment]
@Gala-G please action the rest of LFW's comments prior to merge

**Comment by Gala-G** [comment]
@LongForWisdom @patrick-j-govalpha 

I discussed some of these points with Long on Monday. I apologize for this oh-so-messy PR!

**Whitespaces:** I have no idea why there are so many unwanted whitespaces. I was unable to delete the lines or revert them. MIP41c5-SP2 - which should not be touched in this PR - was part of this whitespace problem. I will start being careful about this.

**SPs in "Formal Submission" marked as "Obsolete":** I was able to verify most of them and marked them as "Accepted" based on our discussion with LFW. I was **unable** to trace down MIP10c9-SP16 and MIP10c7-SP1. Should I mark them back as "Formal Submission", do you know where I can find if they were ratified? Let me know please.

MIP14c2-SP5 that I missed on the initial commit, was done.

Let me know if there is anything else I should do
Thank you!

**Comment by ghost** [comment]
> Whitespaces: I have no idea why there are so many unwanted whitespaces.

I suspect your text editor is configured to trim whitespace. Which editor are you using?

**Pull request merged**